src/tools/genpolicy/update_policy_samples.py

```diff
@@ -11,6 +11,7 @@
     "configmap/pod-cm2.yaml",
     "deployment/deployment-azure-vote-back.yaml",
     "deployment/deployment-azure-vote-front.yaml",
+    "deployment/deployment-busybox.yaml",
```

Comment:
Deploying these non-acr and non-mcr hosted containers causes s360 alerts on our clusters.